outdated/modules/httpd.reb

```diff
@@ -225,199 +256,221 @@ sys/make-scheme [
         close?: true
         compress?: false
 
-        render: func [response [string! binary!]][
```

Comment:
Non-Stylistic change: a plain FUNC would bind to the base object and never be forwarded.  METHOD will bind into the object (found here e.g. by quoting the RENDER: set-word! on it left, and getting its binding) and thus establish a relationship saying that derivations should forward references in that object to their own sub-instance.

This is how derived binding works, and how each object instance does not need to copy METHOD bodies.  *Which is a very good thing.*